cli/test/README.md

```diff
@@ -560,6 +560,53 @@ This document includes CLI commands and the expected CLI output in order to help
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].kind udp_v4`|N/A|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].kind other`|`Fast DDS QoS Profiles Manager exception caught: value 'other' not in enumeration`|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].listening_port`|**PENDING** (currently `Listening port configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].logical_port_increment`|**PENDING** (currently `Logical port increment configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].logical_port_range`|**PENDING** (currently `Logical port range configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].max_initial_peers_range`|**PENDING** (currently `Maximum initial peers range configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].max_logical_port`|**PENDING** (currently `Maximum logical port configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].max_message_size`|**PENDING** (currently `Maximum message size configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].non_blocking_send`|**PENDING** (currently `Non-blocking send configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].output_port`|**PENDING** (currently `Output port configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].port_queue_capacity`|**PENDING** (currently `Port queue capacity configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].receive_buffer_size`|**PENDING** (currently `Receive buffer size configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].rtps_dump_file`|**PENDING** (currently `RTPS dump file configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].segment_size`|**PENDING** (currently `Segment size configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].send_buffer_size`|**PENDING** (currently `Send buffer size configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].tls`|**PENDING** (currently `TLS configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].ttl`|**PENDING** (currently `TTL configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set transport_descriptor[transport_id].wan_address`|**PENDING** (currently `WAN address configuration not yet supported`)|N/A|
```

Comment:
I am missing the following test case `fastddsqosprof file.xml set transport_descriptor[transport_id].other`.